src/fusor/models.py

```diff
@@ -112,6 +116,70 @@ class BaseStructuralElement(ABC, BaseModel):
     type: StructuralElementType
 
 
+class BreakpointCoverage(BaseStructuralElement):
+    """Define BreakpointCoverage class.
+
+    This class models breakpoint coverage, or the number of fragments
+    that are retained near the breakpoint for a fusion partner
+    """
+
+    type: Literal[FUSORTypes.BREAKPOINT_COVERAGE] = FUSORTypes.BREAKPOINT_COVERAGE
+    fragmentCoverage: int
+
+    model_config = ConfigDict(
+        json_schema_extra={
+            "example": {"type": "BreakpointCoverage", "fragmentCoverage": 180}
+        }
+    )
+
+
+class ContigSequence(BaseStructuralElement):
+    """Define ContigSequence class.
+
+    This class models the assembled contig sequence that supports the reported fusion
+    event
+    """
+
+    type: Literal[FUSORTypes.CONTIG_SEQUENCE] = FUSORTypes.CONTIG_SEQUENCE
+    contig: str
```

Comment:
Are any characters allowed? If not, we should enforce a regex pattern: https://docs.pydantic.dev/latest/api/types/#pydantic.types.constr